src/javascripts/ng-admin/es6/lib/DataStore/DataStore.js

```diff
@@ -0,0 +1,119 @@
+import Entry from "../Entry";
+
+class DataStore {
+    constructor() {
+        this._entries = {};
```

Comment:
We tried that, unfortunately babel doesn't transpile ES6 Sets. It requires a shim and/or works only in latest Chrome versions.